src/App.css

```diff
@@ -1,38 +1,272 @@
+
+.clearfix:after {visibility: hidden; display: block; font-size: 0; content: ''; clear: both; height: 0; }
+
+html { -moz-box-sizing: border-box; -webkit-box-sizing: border-box; box-sizing: border-box;}
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+
+html {
+  font-size: 125%;
+  background-color: whitesmoke;
+}
+
+body, 
+.results p {
+  font-family: 'Comfortaa', cursive;
+}
+
+.wrapper {
+  margin: 0 auto;
+  max-width: 1260px;
+  width: 80%;
+}
+
 .App {
   text-align: center;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+h1 {
+  font-size: clamp(1.6rem, 8vw, 4.4rem);
+  font-weight: 700;
+  margin-top: 40px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.title, 
+a {
+  animation: color-animation 7s linear infinite;  
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.firstColor {
+  --color-1: #79cc74;
+  --color-2: #3D8DAE;
+  --color-3: #813aa3;
+  --color-4: #ef950c;
+  --color-5: #c31f9a;
+}
+
+.secondColor {
+  --color-1: #5f67d0;
+  --color-2: #7fc1ba;
+  --color-3: #bd5500;
+  --color-4: #8b95ff;
+  --color-5: #196c72;
+}
+
+@keyframes color-animation {
```

Comment:
i love this animation!  its so subtle and not distracting and so cool at the same time